klaas/scripts/split_data.py

```diff
@@ -0,0 +1,55 @@
+import click
+import numpy as np
+
+from ..io import read_data
+import warnings
+
+
+@click.command()
+@click.argument('input_path', type=click.Path(exists=True, file_okay=True))
+@click.argument('output_basename')
+@click.option(
+    '--fraction', '-f', multiple=True, type=float,
```

Comment:
tried to use a fraction of 0.1 but it did not work : `Error: Invalid value for "--fraction" / "-f": mt is not a valid floating point value` not sure what one has to give here